### ISSUE TITLE
[XamarinProfiler-MAC]: Accessibility: Voiceover: MAS4.2.1 : The positions of the 'Every' and 'On garbage collection' radio buttons under 'Allocations' in the 'Start Profiling' page are not announced by the VoiceOver.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#437]
### Prerequisites

1. Install Xamarin Profiler.
2. Launch Visual Studio.
3. Create a project and build the solution.
4. Navigate to Run Menu Tab and Invoke it. Navigate to "Start Profiling" option and invoke it.
5. This will invokes "Xamarin Profiler" application."
6. Navigate to the "Choose Target" button of the Xamarin Profiler window.

### Steps to Reproduce

1. Follow the pre-requisite to launch the xamarin profiler application.
2. Select the Available instruments to "Allocations/Time Profiler" and invoke next button.
3. In the pane, under 'Allocations' , observe the behavior of the VoiceOver for the 'Every'/' On group collection' radio buttons.

### Expected Behavior

The VoiceOver should announce the positions of the 'Every'/'On garbage collection' radio button.

### Actual Behavior

The VoiceOver is announcing just the names of the radio buttons and not their respective positions.

### User Impact

The screenreader users will not understand how many radio buttons are present under 'Allocations' if the positions are not announced.

### Screenshots

![voiceover_profilermac9](https://user-images.githubusercontent.com/22956130/41715215-55af7952-7570-11e8-865a-4779e78f3ebb.png)

### Environment

macOS High Sierra OSX version: 10.13.4
Visual Studio for Mac Enterprise version: 7.5.2(build 40)
Xamarin Profiler version: 1.6.3-400

**Comment by WiproPSAccTest** [comment]
Please add the following tags to this bug:
A11yMAS
Accessibility
MAS4.2.1
Xamarin-Profiler-MAC
Wipro
